app/controllers/users_controller.rb

```diff
@@ -1,17 +1,24 @@
 class UsersController < ApplicationController
   def show
-      @user = User.find(params[:userID])
+      @user = User.find(params[:id])
+      @faveitems = Array.new
+      @listeditems = Item.all.where("listed==?", true).where("user_id==?", @user.id)
+      @unlisteditems = Item.all.where("listed==?", false).where("user_id==?", @user.id)
   end
   
   def edit
     @user = User.find(params[:id])
+    if(@user != current_user)
```

Comment:
Trying to think of a scenario where this could happen. Good to have that check anyways tho

Reply:
I think maybe if the user changes the URL to edit a user for another user's ID, like edit/user/2 instead of 1